stoic_web/website/views.py

```diff
@@ -1,10 +1,15 @@
 import random
 from django.shortcuts import render
-from django.views.generic import TemplateView, ListView, DetailView
+from django.views.generic import TemplateView, ListView, DetailView, CreateView
```

Comment:
Yes - it seems the Django `CreateView` was never actually being used, so I'll remove this